### DISCUSSION TITLE
Oqtane v2.0.1

### DISCUSSION THREAD

**Discussion by sbwalker** [oqtane/oqtane.framework#1144] [General]
I just published the 2.0.1 release - https://github.com/oqtane/oqtane.framework/releases/tag/v2.0.1. The list of changes in the release is lengthy and I very much appreciate the contributions, whether they are are PR's, issue reports, or just general discussion items on how to improve the product. I am still writing the formal release announcement... it will focus on the work done by @hishamco to complete the Localization support. Now that 2.0.1 is published I would hope to see some activity in the https://github.com/oqtane/oqtane.translations repo :)

**Comment by ADefWebserver** [comment]
Please see: https://github.com/oqtane/oqtane.framework/issues/1145

**Comment by hishamco** [comment]
Sure the translations is the upcoming thing which I'm working on ...

**Comment by michalss** [comment]
I would translate it into my language, but is there any res file or any other file to translate ? In that repo, nothing there..

**Reply by hishamco** [reply]
@michalss please click the "Watch" button on the translation repo, they will be show up in upcoming days ...

**Comment by sbwalker** [comment]
Release blog: https://www.oqtane.org/Resources/Blog/PostId/546/oqtane-201-now-available

**Reply by sbwalker** [reply]
@hishamco do you have any idea when you might be able to submit some language resources to the Translation repo so that people can understand how to get started with localization? I have managed to get confirmation to be a guest on the .NET Community Standup in April and I would really like to showcase the localization support but I cant at this point because I do not have any language resources.

**Reply by ADefWebserver** [reply]
> I have managed to get confirmation to be a guest on the .NET Community Standup in April

@sbwalker Do we have a link yet? We need to promote this!

**Reply by sbwalker** [reply]
It is scheduled for Tuesday, April 13, 2021 9:45 AM-11:15 AM PT - however there is no link yet to promote.

**Reply by hishamco** [reply]
@sbwalker I'm working on an extraction tool, I will try to update the work I'm doing in the repo https://github.com/hishamco/Oqtane.ResxExtractor I already started on it

I will try to publish `Arabic` resources once it's done, hope to finalize the work on that tool within this weeks. Once it's done it's much easier to add other languages. If there's anything you need me to do please let me know via GitHub or Gitter

FYI I have few other OSS I'm working on, but I will try to give some priority to Oqtane for sure

**Reply by hishamco** [reply]
@sbwalker glad to inform you that the tools works well, I already generate the default resources and add them to `Oqtane.Translations` repo. My next steps to ensure that everything as expected, also I will start add Arabic resources as proof for another language 

Also I will add some instructions to ensure that the app localized

**Reply by FrancoisMongeau** [reply]
I'd be glad to help for French. Will you provide some guidance?

**Reply by hishamco** [reply]
@FrancoisMongeau thanks, you could visit Oqtane.Translations repo, I will try to write some instructions, but basically you could start with the available RESX files for English and translate them. Hope to do the same thing for Arabic language

**Reply by pcancela** [reply]
@hishamco I can help with Portuguese. Correct me if I'm wrong, I'm supposed to download your https://github.com/hishamco/Oqtane.ResxExtractor and then run and generate those resx files and apply the translations?

**Reply by hishamco** [reply]
@pcancela thanks, we could move our discussion in Oqtane.Translation. BTW you could clone and run the tool that I made, but it will produce the same result ;)

I'm thiking to integrate this tool later with GitHub Actions, so it can run if the Oqtane code has been changed or think for a better way to ensure that translation are FRESH

**Reply by michalss** [reply]
I will do Czech.. :) Also would like know the procedure how to do it ?

**Reply by hishamco** [reply]
> Also would like know the procedure how to do it ?

Please check the documented steps that I added few minutes ago in https://github.com/oqtane/oqtane.translations
